pkg/template/template.go

```diff
@@ -48,10 +48,10 @@ func (t *Template) Err() error {
 // Execute parses the provided text (or returns it unchanged if not a Go template), associates it with the templates
 // defined in t.tmpl (so they may be referenced and used) and applies the resulting template to the specified data
 // object, returning the output as a string.
-func (t *Template) Execute(text string, data interface{}) string {
-	log.V(2).Infof("Executing template %q...", text)
+func (t *Template) Execute(text string, data interface{}, logger log.Logger) string {
+	level.Debug(logger).Log("msg", "executing template", "template", text)
 	if !strings.Contains(text, "{{") {
-		log.V(2).Infof("  returning unchanged.")
+		level.Debug(logger).Log("msg", "returning unchanged template")
```

Comment:
Same as above, these log messages are intended for debugging only, they go after one another in the log so they are easier to eyeball if they are simpler and the follow-up messages are indented.